Gemfile.lock

```diff
@@ -597,12 +599,12 @@ GEM
     rubocop-rails (2.3.2)
       rack (>= 1.1)
       rubocop (>= 0.72.0)
-    ruby-progressbar (1.10.1)
+    ruby-progressbar (1.13.0)
     ruby2_keywords (0.0.5)
-    ruby_parser (3.13.1)
-      sexp_processor (~> 4.9)
+    ruby_parser (3.21.0)
```

Comment:
### [ruby_parser](https://github.com/seattlerb/ruby_parser)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.13.1 → 3.21.0

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/ruby_parser/3.13.1/3.21.0)_]





<details>
<summary>Commits</summary>

A change of **212** commits. See the full changes on [the compare page](https://github.com/seattlerb/ruby_parser/compare/v3.13.1...v3.21.0).

These are the first 10 commits:
- (3ddf19e) [- Fixed differentiation of `{ |a| ... }` vs `{ |a,| ... }`. …](https://github.com/seattlerb/ruby_parser/commit/3ddf19e877c0e3df0766214c79bc7b473c56af81)
- (61aa2df) [Added better error handling/reporting for tools/ripper.rb](https://github.com/seattlerb/ruby_parser/commit/61aa2dfeabac006576bd77962711d3a85c80efaa)
- (e40bc36) [- Differentiated between lambda call and stabby proc. Added …](https://github.com/seattlerb/ruby_parser/commit/e40bc36ad0af6415f917a22e21d6f6ddbf4b2206)
- (9441a89) [- Fixed parsing of () in a command-call (eg p).](https://github.com/seattlerb/ruby_parser/commit/9441a89aa366265ca08dc0cd16545f69ea2c0195)
- (70d5a88) [Fixed munging newer ruby's 'next token' output](https://github.com/seattlerb/ruby_parser/commit/70d5a8849daf8169354a08cae5809234ecb99f94)
- (1bca3b2) [Made rake debug and debug3 default to F=bug.rb](https://github.com/seattlerb/ruby_parser/commit/1bca3b278d3c2b0a691d26a815915c9a149bfa26)
- (ab2dfce) [- Fixed some lex_state versioning issues with closing braces…](https://github.com/seattlerb/ruby_parser/commit/ab2dfcea909602c3d7170a49127918ec371623ee)
- (9d1243b) [Patched up .autotest to be more resilient.](https://github.com/seattlerb/ruby_parser/commit/9d1243b40fbcdd8ab59781a73c52db963c5a09c2)
- (b594974) [- Fixed lex_state for tSTRING_END and tREGEXP_END.](https://github.com/seattlerb/ruby_parser/commit/b594974d1487cfba09f092d73d74e9d8ebed48cc)
- (83a340b) [- Extended State#== to check equal? first as an optimization…](https://github.com/seattlerb/ruby_parser/commit/83a340b5e91a23b847be6c08c8d4e1505ac07180)


</details>